processor/processor.go

```diff
@@ -433,6 +437,11 @@ func (proc *Handle) Setup(
 		proc.logger.Errorn("failed to create ut sampling file manager", obskit.Error(err))
 		proc.utSamplingFileManager = nil
 	}
+	proc.storeSamplingFileManager, err = getStoreSamplingUploader(proc.conf, proc.logger)
+	if err != nil {
+		proc.logger.Errorn("failed to create store sampling file manager", obskit.Error(err))
+		proc.storeSamplingFileManager = nil
```

Comment:
Why do we need to explicitly mention it as nil? Doesnt the function return nil when an error occurs?

Reply:
Removed it. Copied code from the UT sampling file manager.